docs/authentication/social-connections/huggingface.mdx

```diff
@@ -0,0 +1,56 @@
+---
+title: Hugging Face
+description: Learn how to set up social connection with Hugging Face.
+---
+
+# Hugging Face
+
+How to set up social connection with Hugging Face
+
+## Overview
+
+Adding social connection with Hugging Face to your app with Clerk is done in a few steps - you only need to set the **Client ID**, **Client Secret** and **Authorized redirect URI** in your instance settings.
+
+To make the development flow as smooth as possible, Clerk uses preconfigured shared OAuth credentials and redirect URIs for development instances - no other configuration is needed.
+
+For production instances, you will need to create your own Client ID and Client secret using your Hugging Face account.
+
+> The purpose of this guide is to help you create a Hugging Face Connected App - if you're looking for step-by-step instructions using Clerk to add social connection (OAuth) to your application, follow the [Social connection (OAuth) guide](/docs/authentication/social-connections/oauth).
+
+## Before you start
+
+- You need to create a Clerk Application in your [Clerk Dashboard](https://dashboard.clerk.com/). For more information, check out our [Set up your application guide](/docs/quickstarts/setup-clerk).
+- You need to have a Hugging Face account. To create one, [click here](https://huggingface.co/login).
+
+## Configuring Hugging Face social connection
+
+First, you need to create a new Connected App. Navigate to **Settings > Connected Apps** and click on **Create application button**. On the following screen fill all the required information, such as the name, the logo etc.
+
+In the **Scopes** field its important to select at least _openid_, _profile_ and _email_.
+
+<Images
+  width={1904}
+  height={981}
+  src="/docs/images/authentication-providers/huggingface/create-application.png"
+  alt="Creating a new application"
+/>
```

Comment:
I think you should replace `Images` component with 
```
![Creating a new application](/docs/images/authentication-providers/huggingface/create-application.png)
```

Reply:
Yep updated thanks!